spec/commands/update_spec.rb

```diff
@@ -809,6 +809,15 @@
       G
     end
 
+    context "with patch set as default update level in config" do
+      it "should do a patch level update" do
+        bundle! "config --local patch true"
+        bundle! "update --patch foo"
```

Comment:
This shouldn't include the `--patch`, right?

```suggestion
        bundle! "update foo"
```

Reply:
Yes, corrected that. :sweat_smile: